test/hotspot/jtreg/compiler/intrinsics/unsafe/HeapByteBufferTest.java

```diff
@@ -35,6 +35,18 @@
  *      HeapByteBufferTest
  */
 
+/**
+ * @test
+ * @bug 8282555
+ * @summary intermittent, check that spilling MoveF2I etc produce memory edge
+ * @modules java.base/jdk.internal.misc
+ * @library /test/lib
+ *
+ * @run main/othervm -Djdk.test.lib.random.seed=42
```

Comment:
Please add an additional run without a fixed seed.

Reply:
@TobiHartmann Done and tested

Reply:
Thanks!